### DISCUSSION TITLE
Made Changes to my profile , Your Suggestions would be helpful

### DISCUSSION THREAD

**Discussion by AakarshTeja** [EddieHubCommunity/support#151] [Show and tell]
**Made Changes to my profile, Your Suggestions would be helpful
https://github.com/AakarshTeja**

![image](https://user-images.githubusercontent.com/42816874/92401995-728eef80-f14c-11ea-914c-5aec16fea541.png)

**Comment by eddiejaoude** [comment]
Looks good 👍 , some of my feedback:

- I think to the right of the stats card, you could make better use of that blank space
- List of skills might look better as a vertical float left - not sure, you will have to try this one and see

**Comment by BOLT04** [comment]
Hey @AakarshTeja, awesome profile 👍. I saw a couple missing spaces, like "University.I am"  and "Aboutme", unless those are intented 🤓.
A little suggestion could be to have a link to the project where you used a given skill, instead of going to the image. Other than that, it's pretty cool 😃.

**Reply by AakarshTeja** [reply]
Thank you for the valuable feedback bro😍. I will fix them and I will try to add links for my projects for the skills 😉. Any other suggestions will also be welcome😁🤝.

**Comment by Jin-Xu** [comment]
Awesome stuff, since my is mostly empty at the moment I will be stealing some stuff from yours. :)

**Reply by eddiejaoude** [reply]
Good idea, I think everyone is borrowing parts they like from different people and using it as inspiration 👍 

Also dont forget we have this repo for inspiration and ideas https://github.com/EddieJaoudeCommunity/awesome-github-profiles
